docs/attributes-registry/db.md

```diff
@@ -163,14 +163,15 @@ This group defines attributes for Cassandra.
 
-| Attribute                            | Type   | Description                       | Examples                               | Stability                                                        |
-| ------------------------------------ | ------ | --------------------------------- | -------------------------------------- | ---------------------------------------------------------------- |
-| `db.cosmosdb.client_id`              | string | Unique Cosmos client instance id. | `3ba4827d-4422-483f-b59f-85b74211c11d` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `db.cosmosdb.connection_mode`        | string | Cosmos client connection mode.    | `gateway`; `direct`                    | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `db.cosmosdb.operation_type`         | string | Cosmos DB Operation Type.         | `batch`; `create`; `delete`            | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `db.cosmosdb.request_charge`         | double | RU consumed for that operation    | `46.18`; `1.0`                         | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `db.cosmosdb.request_content_length` | int    | Request payload size in bytes     |                                        | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `db.cosmosdb.sub_status_code`        | int    | Cosmos DB sub status code.        | `1000`; `1002`                         | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| Attribute                            | Type   | Description                           | Examples                                                                | Stability                                                        |
+| ------------------------------------ | ------ | ------------------------------------- | ----------------------------------------------------------------------- | ---------------------------------------------------------------- |
+| `db.cosmosdb.client_id`              | string | Unique Cosmos client instance id.     | `3ba4827d-4422-483f-b59f-85b74211c11d`                                  | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `db.cosmosdb.connection_mode`        | string | Cosmos client connection mode.        | `gateway`; `direct`                                                     | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `db.cosmosdb.consistency_level`      | string | Account or request consistency level. | `Eventual`; `ConsistentPrefix`; `BoundedStaleness`; `Strong or Session` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `db.cosmosdb.operation_type`         | string | Cosmos DB Operation Type.             | `batch`; `create`; `delete`                                             | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
I was under the impression that we're going to remove `operation_type` - it's ok to do it in  a separate PR (or in this one - however you prefer to do it)

Reply:
Initially, I had planned to remove it in a separate PR, but since we've decided not to modify the attributes and only removing `operation_type`, I'm proceeding with this change in the current PR